app/controllers/bids_controller.rb

```diff
@@ -61,4 +63,22 @@ def is_owner_of_bid?
       redirect_to root_path
     end
   end
+
+  def ensure_less_then_ten_bids?
+    @bids = Bid.where(bidder_id: current_user.id, bidder_type: current_user.class.name)
+    @bids_this_month = @bids.this_month.count
+    if @bids_this_month >= 10 && current_user.account_type == 0 && broker_signed_in?
+      flash[:alert] = "Sorry! You can't bid on more then 10 jobs please switch to premium plan."
+      redirect_to root_path
+    end
+  end
+
+  def ensure_less_then_fifteen_bids?
+    @bids = Bid.where(bidder_id: current_user.id, bidder_type: current_user.class.name)
+    @bids_this_month = @bids.this_month.count
+    if @bids_this_month >= 1 && current_user.account_type == 0 && forward_freight_signed_in?
+      flash[:alert] = "Sorry! You can't bid on more then 15 jobs please switch to premium plan."
+      redirect_to root_path
+    end
+  end
```

Comment:
Oh yeah, and then you can use the `bid_limit_exceeded?` in the model validation as well.